spec/cases/html_page_spec.rb

```diff
@@ -46,15 +46,15 @@
 <font face="微软雅黑 "><font size="3"><strong>DIY的手绘行程图</strong></font></font><br>
 <span style="position: absolute; display: none" id="attach_1254048" onmouseover="showMenu({'ctrlid':this.id,'pos':'13'})"><img src="images/go2eu/attachimg.gif" border="0"></span>
 <img src="http://att.qyer.com/day_111228/1112282241c0deb3a69dcd9cf5.jpg" file="http://att.qyer.com/day_111228/1112282241c0deb3a69dcd9cf5.jpg" width="700" class="zoom" onclick="zoom(this, this.src)" id="aimg_1254048" onmouseover="showMenu({'ctrlid':this.id,'pos':'12'})" alt="55105222201110032253301023057594630_006.jpg">
-<div class="t_attach" id="aimg_1254048_menu" style="position: absolute; display: none">				
+<div class="t_attach" id="aimg_1254048_menu" style="position: absolute; display: none">
```

Comment:
my editor stripped whitespace. Sorry! If you'd like me to unstage these commits or anything so it's not impacted, just let me know.